src/ArraysAveragerTest.java

```diff
@@ -0,0 +1,28 @@
+import org.junit.jupiter.api.Test;
+
+import static java.lang.Double.NaN;
+import static org.junit.jupiter.api.Assertions.*;
+
+class ArraysAveragerTest {
+
+    @Test
+    void arraysAverager() {
```

Comment:
Consider giving your tests more informative names, like `tinyArrAverager` for this one, `jaggedArrAverager` for the next one, and `emptyArrAverager` for the last one.  Then you will know immediately which ones are passing/failing, which is hugely useful when you have dozens of tests running in parallel.